src/main/java/com/hu/webdriver/pages/message/MessagePage.java

```diff
@@ -0,0 +1,207 @@
+	 */
+	public void clickOnCompose() {
+		findClickableElement(By.cssSelector(".icon-pencil-1")).click();
+	}
+
+	/**
+	 * Method to click on Delete message.
+	 */
+	public void clickOnDeleteMessage() {
+		findClickableElement(By.xpath("//span[text()='Delete']")).click();
+		sleep(4);
+	}
+
+	/**
+	 * Method to click on Mail.
+	 */
+	public void clickOnMail() {
+		findClickableElement(By.xpath("(//input[@type='checkbox'])[11]/following::div[2]/h5//span/strong")).click();
+	}
+
+	/**
+	 * Method to click on Messages.
+	 */
+	public void clickOnMessages() {
+		findClickableElement(By.xpath("//span[text()='Messages ']")).click();
+	}
+
+	/**
+	 * Method to click on Select message.
+	 */
+	public void clickOnSelectMessage() {
+		findClickableElement(By.xpath("(//input[@type='checkbox'])[11]")).click();
+	}
+
+	/**
+	 * Method to click on Send message.
+	 */
+	public void clickOnSendMessage() {
+		findClickableElement(By.cssSelector(".btn-primary")).click();
+		sleep(1);
+	}
+
+	/**
+	 * Method to click on Undo.
+	 */
+	public void clickOnUndo() {
+		findClickableElement(By.xpath("//*[text()='Undo?']")).click();
+		sleep(3);
+	}
+
+	/**
+	 * Method to Current url.
+	 *
+	 * @return String.
+	 */
+	public String currentUrl() {
+		return driver.getCurrentUrl();
+	}
+
+	/**
+	 * Method to verify whether Mail displayed.
+	 *
+	 * @return boolean.
+	 */
+	public boolean isMailDisplayed() {
+		return isElementPresent(mail);
+	}
+
+	/**
+	 * Method to verify whether Mail title displayed.
+	 *
+	 * @return boolean.
+	 */
+	public boolean isMailTitleDisplayed() {
+		return isElementPresent(By.xpath("(//input[@type='checkbox'])[11]/following::div[2]/h5//span/strong"));
+	}
+
+	/**
+	 * Method to verify whether mark as read displayed.
+	 *
+	 * @return boolean.
+	 */
+	public boolean isMarkAsReadDisplayed() {
+		return isEnabled(By.xpath("//button[text()='Mark as read']"));
+	}
+
+	/**
+	 * Method to verify whether Message displayed.
+	 *
+	 * @param userName.
+	 * @param userNameTextbana.
+	 * @return boolean.
+	 */
+	public boolean isMessageDisplayed(String userName, String userNameTextbana) {
+		return isElementPresent(
+				By.xpath("(//strong[text()='Message between " + userName + " and " + userNameTextbana + "'])[2]"));
+	}
+
+	/**
+	 * Method to verify whether Replay mail displayed.
+	 *
+	 * @return boolean.
+	 */
+	public boolean isReplayMailDiplayed() {
+		return isElementPresent(By.xpath("(//input[@type='checkbox'])[11]/following::div[2]/h5//span/strong"));
+	}
+
+	/**
+	 * Method to verify whether Undo displayed.
+	 *
+	 * @return boolean.
+	 */
+	public boolean isUndoDisplayed() {
+		sleep(3);
+		return isElementPresent(undo);
+	}
+
+	/**
+	 * Method to navigate to Message page.
+	 */
+	public void navigateToMessagePage() {
+		driver.navigate().to("https://healthunlocked.com/messages");
+	}
+
+	/**
+	 * Method to set Message text box.
+	 *
+	 * @param value.
+	 */
+	public void setMessageTextBox(String value) {
+		messageTextBox.sendKeys(value);
```

Comment:
use clear......
or
Instead of it create one generic method on BasePage

public void clearAndType(WebElement element,String value){
    element.clear();
    element.sendKeys(value);
}